landoapi/commit_message.py

```diff
@@ -39,7 +39,7 @@
 
 # Note that we only allows a subset of legal IRC-nick characters.
 # Specifically, we do not allow [ \ ] ^ ` { | }
-IRC_NICK = r"[a-zA-Z0-9\-\_]+"
+IRC_NICK = r"[a-zA-Z0-9\-\_.]+[a-zA-Z0-9\-\_]+"
```

Comment:
The PR title says the patch enables author names with a single period.  However I got some pretty funky names with multiple periods to match the regex, too.  Is that behaviour OK?

```
aa
a.b
a.b.c
.a
...a
a...a
-.-.-
```

Reply:
Yes - we need to support multiple periods in names.

Reply:
Indeed supporting multiple periods would be good. I originally had this in my approach but simplified the regex to match a single period however we might as well support an arbitrary number of periods from the get go. Will update.